skaha/src/main/webapp/service.yaml

```diff
@@ -41,7 +41,7 @@ paths:
       - name: view
         in: query
         type: string
-        description: Session list view level.  If set to 'all', all users' sessions are listed (with limited information), not just the calling user's sessions.
+        description: Session list view level and stats.  If set to 'all', all users' sessions are listed (with limited information), not just the calling user's sessions. If set to 'stats', jobs and resources (cores and ram) information is listed.
```

Comment:
While you are making changes in here, could you please correct the github URLs above?  Should be 'science-platform' instead of 'skaha'.